app/screens/webview/webview.tsx

```diff
@@ -45,17 +50,36 @@ export const WebViewScreen: React.FC<Props> = ({ route }) => {
   React.useEffect(() => {
     navigation.setOptions({
       headerLeft: () => (
-        //                                  FIXME < is not the same as for other screens
-        <Button onPress={handleBackPress} title={`< ${LL.common.back()}`} />
+        <TouchableOpacity style={styles.iconContainer} onPress={handleBackPress}>
+          <GaloyIcon
+            name="caret-left"
+            size={20}
+            color={mode === "light" ? colors.black : colors._white}
```

Comment:
```suggestion
            color={colors.black}
```

the `_white` is a hardcoded color to while, while the `black` is a color that would be different depending on the theme. it would be black for the light mode, but white for the dark mode. 

I think just having `colors.black` would just work the same, with the benefits of been clearer on the intent.